### DISCUSSION TITLE
Confirmation of Project Status (seeing In Progress)

### DISCUSSION THREAD

**Discussion by AmanPriyanshu** [OpenMined/courses#225] [Project: Privacy Product Specification] [answered]
I am glad to have done this course and learn so many amazing things from it. 
With regards to the final project, I submitted my Project last week (8 days ago). However, I am not sure whether the system seems to have accepted my submission. On the project tab (https://courses.openmined.org/courses/our-privacy-opportunity/project), I can see the status `"In Progress"` (not `"under review"`). I am just worried that I may not have followed the steps correctly and am waiting for no reason without having submitted any project.

Even so, at the bottom I have gotten the message:
`Thank you for your submission! Out mentors will review your work and give you feedback within 1-2 days. You will receive a notification and an email when your project has been reviewed. In the meantime, see what others did by discussing this project on our Discussion Board.`

So I am completely unsure of the status of my project. I would love to correct my mistake and get my project on the right track.

**Comment by yemikifouly** [comment, marked as answer]
Hello Aman! 👋 There is absolutely no reason to worry. As long as you got the message that you mention your project was successfully submitted and you will receive feedback as soon as our mentors review it. 🙂

**Reply by AmanPriyanshu** [reply]
Thanks a lot!
